litmus-portal/backend/graphql-server/pkg/graphql/subscriptions/subscription.go

```diff
@@ -34,3 +36,24 @@ func SendWorkflowEvent(wfRun model.WorkflowRun, r store.StateData) {
 	}
 	r.Mutex.Unlock()
 }
+
+func SendWorkflowRequest(wfRequest database.ChaosWorkFlowInput, r store.StateData) {
+
+	newAction := model.ClusterAction{
+		ProjectID: wfRequest.ProjectID,
+		Action: &model.ActionPayload{
+			K8sManifest: wfRequest.WorkflowManifest,
+			RequestID:   rand.Int(),
```

Comment:
Having an RequestId that is not being used doesn't makse sense? We are not storing it anywhere?

Reply:
Yes, No need of requestedid for now. I have removed it